lib/executables_db.rb

```diff
@@ -61,7 +61,7 @@ def update!(update_existing: false, install_missing: false, max_downloads: nil,
       taps = eval_all ? Tap.each.to_a : [CoreTap.instance]
```

Comment:
```suggestion
      taps = eval_all ? Tap.select(&:installed?) : [CoreTap.instance]
```

to handle https://github.com/Homebrew/brew/pull/16710 properly.